lib/ReactTags.js

```diff
@@ -75,6 +78,7 @@ class ReactTags extends Component {
       selectionMode: false,
     };
 
+    this.addTag = this.addTag.bind(this);
```

Comment:
why binding here ?

Reply:
To prevent issues when calling this.props

Reply:
you can use arrow functions in that case :)